### PR TITLE
Updated project to better support Android Studio

### PR DESCRIPTION
**Pull request by ankurkotwal**
* Migrated to Android Studio project structure
* Updated Google Play Services dependency to google-play-services-maps:8.4.0
* Removed reference to support library
* Removed references to Eclipse from README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,47 +3,37 @@ A fast way to get started with the [Google Maps Android API v2](http://developer
...
+## Open the project
+  1. On the welcome screen, select "Open an existing Android Studio project" or if you have a project open, choose "File->Open...""
   2. Choose Android > Existing Android Code Into Workspace
   3. Navigate to the checked out project
-  4. You should see two projects - "hellomap" and "google-play-services_lib".
-  5. Ensure both are selected
-  6. Press "Finish"
+  4. Press "Ok"
 
 ## Add your API key
```
Comment (jfschmakeit) [inline README.md]: This section doesn't look right for Android Studio. in the default 'Android' view, the manifest is under 'Application -> manifests'

File: README.md [hunk 2]
```diff
@@ -53,39 +43,8 @@ Eclipse is packaged with the [Android Developer Tools bundle](http://developer.a
...
-  3. Press the + button
-  4. Click "Android Application"
-  5. Edit the name
-  6. Choose "hellomap" as the Module
-  7. Choose "USB Device" as your Target Device
-  8. Press OK
-  9. Press the green play button next to the name you specified. This will build the app, install it on the device and run it.
-
-## Troubleshooting
 ### "Please specify Android SDK"
```
Comment (jfschmakeit) [inline README.md]: I don't think this error can occur in Android Studio. Either way, the instructions don't work.

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.2"
+
+    defaultConfig {
+        applicationId "com.example.hellomap"
+        minSdkVersion 9
+        targetSdkVersion 17
```
Comment (jfschmakeit) [inline app/build.gradle]: targetSdkVersion = 23 ?

File: app/src/debug/res/values/google_maps_api.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+
+    To get one, follow this link, follow the directions and press "Create" at the end:
+
+    https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=B5:C0:79:09:26:87:04:DF:F7:17:37:5A:B5:71:54:15:F7:A4:CA:A2%3Bcom.sample.hello
```
Comment (jfschmakeit) [inline app/src/debug/res/values/google_maps_api.xml]: Remove this link and the line below - we shouldn't be checking in a debug key

File: build.gradle [hunk 5]
```diff
@@ -1,30 +1,23 @@
...
-    buildToolsVersion "17.0.0"
-
-    sourceSets {
-        main {
-            manifest.srcFile 'AndroidManifest.xml'
-            java.srcDirs = ['src']
-            res.srcDirs = ['res']
-        }
-    }
+task clean(type: Delete) {
```
Comment (jfschmakeit) [inline build.gradle]: Do we need this?

File: README.md [hunk 6]
```diff
@@ -3,47 +3,36 @@ A fast way to get started with the [Google Maps Android API v2](http://developer
...
-  6. Press "Finish"
+## Open the project
+  1. On the welcome screen, select "Open an existing Android Studio project" or if you have a project open, choose "File->Open...""
+  2. Navigate to the checked out project
+  3. Press "Ok"
 
 ## Add your API key
-  1. Navigate to AndroidManifest.xml, under the "hellomap" project.
+  1. Navigate to the AndroidManifest.xml file
   2. Click the "AndroidManifest.xml" tab, next to "Instrumentation", to see the XML view.
```
Comment (markmcd) [inline README.md]: Is this line still true?

File: README.md [hunk 7]
```diff
@@ -53,54 +42,6 @@ Eclipse is packaged with the [Android Developer Tools bundle](http://developer.a
...
-  1. Click "New..."
-  2. Choose "JDK"
-  3. Navigate to the JDK's home directory. For OS X, this is at `/Library/Java/JavaVirtualMachines/jdk.../Contents/Home`
-  4. Press "Choose"
-  5. If you are prompted with "Set up created SDK on project?", press No.
-  6. Follow the instructions above
-  7. If you are prompted with "Set up created SDK on project?", press Yes.
-
-### I see a grey map
 Follow the instructions under "Add your API key" and try again.
```
Comment (markmcd) [inline README.md]: Perhaps add a line here suggesting that the developer should also keep an eye on the logcat output, as the API will often provide extra information there, e.g. if the cert/key aren't set up correctly.

**Comment by ankurkotwal** [comment]
@markmcd @jfschmakeit Have addressed all code review comments. PTAL.

**Comment by markmcd** [comment]
LGTM

Merge at will

**Pull request merged**